components/markdown/embeds/VideoEmbed.tsx

```
@@ -0,0 +1,69 @@
+
+export const VideoEmbed = ({ type, id, isVisible }: VideoEmbedProps) => {
+  const getEmbedUrl = () => {
+    switch (type) {
+      case 'YOUTUBE':
+        return `https://www.youtube.com/embed/${id}?rel=0&modestbranding=1&playsinline=1&enablejsapi=1&autoplay=0&mute=1&origin=https://skatehive.app`;
+      case 'VIMEO':
+        return `https://player.vimeo.com/video/${id}?autoplay=0&muted=1&origin=https://skatehive.app`;
+      case 'THREESPEAK':
+        return `https://play.3speak.tv/watch?v=${id}&mode=iframe&autoplay=0&muted=1`;
+      case 'ODYSEE':
+        let odyseeBase = '';
```

Biome (2.4.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)